rust/agents/validator/src/settings.rs

```diff
@@ -40,6 +41,8 @@ pub struct ValidatorSettings {
     pub reorg_period: u64,
     /// How frequently to check for new checkpoints
     pub interval: Duration,
+
+    pub staking_config: StakingConf,
```

Comment:
Do you expect to ever have this be set via settings?

if we won't ever have a different config other than the `default_staking_config`, imo you could just not even have this in ValidatorSettings, and just create it in `impl BaseAgent for Validator {`

Reply:
given the above comment, I was anticipating we may need to set it via settings as you mentioned